### DISCUSSION TITLE
Destroying abandoned non persistent zdo

### DISCUSSION THREAD

**Discussion by MapleBZH** [lloesche/valheim-server-docker#192] [Q&A]
Hi,

Is this message in LOG is normal ? : "Destroying abandoned non persistent zdo"
It appears many times...

Thanks in advance for any information

**Comment by lloesche** [comment]
I would think so. I see it all the time as well and haven't noticed any negative effects. Although we had one report where they saw this and their buildings weren't saved.

**Reply by MapleBZH** [reply]
thanks !
any negative effects too...
and thanks for your job with this image ;)
